ui/components/molecules/M007-AdressFormFragment/index.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import createPropTypes from 'json-schema-prop-types'
+
+import Grid from '@material-ui/core/Grid'
+
+import TextFieldLabel from '../../atoms/A010-TextFieldLabel'
+
+import schema from './schema'
+
+const M007 = props => {
```

Comment:
Why is it M007 - we haven't got that high yet? did you have an idea for how you wanted to group things?